lib/puppet/provider/ios_stp_global/cisco_ios.rb

```diff
@@ -61,6 +99,20 @@ def self.mst_commands_from_instance(instance)
     commands
   end
 
+  def self.portfast_commands_from_instance(instance)
+    commands = []
+    instance[:portfast].each do |portfast_option|
```

Comment:
If there is only one `portfast` set then this will be a string and not an array - will cause an error:

```
Error: undefined method `each' for "bpdufilter default":String
```

Reply:
```
Portfast Default             is disabled
PortFast BPDU Guard Default  is disabled
Portfast BPDU Filter Default is enabled
```

Reply:
Sorry these comments were meant for https://github.com/puppetlabs/cisco_ios/pull/312/files#diff-bdada0f552b1434b848fa27e4055597eR44